Extensions/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/src/components/privacySettings/index.jsx

```diff
@@ -251,32 +239,34 @@ class PrivacySettingsPanelBody extends Component {
             </div>
         );
 
-        return (
-            <div className={styles.privacySettings}>
-                <div className="sectionTitle">
-                    {resx.get("PrivacyCommunicationSettings")}
+        if(props.privacySettingsClientModified) {
```

Comment:
I cannot properly test this now since this tab does not load, can you explain the motivation behind this change?